MSA developer support to call app API (#9085)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/application-list.md

Get the list of [applications](../resources/application.md) in this organization.

> NOTE

> > This API would return owned apps if the incoming token is tied to a personal Microsoft account.

Author: Rephrased this sentence. 

@davidmu1 / @dkershaw10 - Do you have a better suggestion?

> > When calling this API using tokens issued for a personal Microsoft account, this API would return the apps owned by the personal Microsoft account. There is no notion of organizations exist for personal Microsoft accounts. In order to list applications owned by specific personal Microsoft accounts this API also requires User.Read permission in addition to Application.Read.All or Application.ReadWrite.All.

Reviewer: ```suggestion
> When calling this API using tokens issued for a personal Microsoft account, this API will return the apps owned by the personal Microsoft account. The notion of organizations doesn't exist for personal Microsoft accounts. In order to list applications owned by specific personal Microsoft accounts this API also requires User.Read permission in addition to Application.Read.All or Application.ReadWrite.All.
```

Reviewer 2: ```suggestion
> When calling this API using tokens issued for a personal Microsoft account, this API returns the apps owned by the personal Microsoft account. Microsoft accounts do not use organizations. In order to list applications owned by specific personal Microsoft accounts this API requires the User.Read permission, and the Application.Read.All or Application.ReadWrite.All permission.
```

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

| Permission type | Permissions (from least to most privileged) |
|:--------------- |:------------------------------------------- |
| Delegated (work or school account) | Application.Read.All, Application.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All    |

> | Delegated (personal Microsoft account) | User.Read.All and Application.Read.All, User.Read.All and Application.ReadWrite.All |

Author: I like the suggestions. Incorporated the change.
